cpp/net/connection_pool.cc

```diff
@@ -32,16 +32,20 @@ using std::unique_ptr;
 using util::ClearOpenSSLErrors;
 using util::DumpOpenSSLErrorStack;
 
-DEFINE_int32(url_fetcher_max_conn_per_host_port, 4,
-             "maximum number of URL fetcher connections per host:port");
+DEFINE_int32(connection_read_timeout_seconds, 60,
+             "Connection read timeout in seconds, only applies while willing to read.");
```

Comment:
That's yoinked from the libevent docs (which is what actually implements it):
http://www.wangafu.net/~nickm/libevent-2.0/doxygen/html/bufferevent_8h.html#a3686d744e67e9a530ebf8ab888ade2cf